src/components/accounts/recovery_setup/SetupRecoveryMethod.js

```diff
@@ -109,10 +132,22 @@ class SetupRecoveryMethod extends Component {
     }
 
     handleSetupRecoveryMethod = async (securityCode) => {
-        const  { accountId, setupRecoveryMessage, redirectToApp } = this.props;
+        const  { accountId, setupRecoveryMessage, redirectToApp, history } = this.props;
+
+        const { error } = await setupRecoveryMessage(accountId, this.method, securityCode)
+
+        if (error) {
```

Comment:
Error handling for promises returned by dispatching Redux actions doesn't work like this anymore:
https://github.com/near/near-wallet/pull/671

Reply:
Fixed: https://github.com/near/near-wallet/pull/700/commits/bb01861fff8dfb496f3987256187b173cbc95d42